tests/phpunit/tests/multisite/getBlogDetails.php

```diff
@@ -61,7 +61,7 @@ public function test_get_blog_details_with_site_name_string_subdomain() {
 			}
 
 			$site = get_blog_details( 'foo' );
-			$this->assertEquals( self::$site_ids[ 'foo.' . WP_TESTS_DOMAIN . '/' ], $site->blog_id );
+			$this->assertSame( self::$site_ids[ 'foo.' . WP_TESTS_DOMAIN . '/' ], $site->blog_id );
```

Comment:
I'm going to skip committing this one. Why? It's not clear to me why `get_block_details()` returns an integer for this one, but not the other tests. I think this test class needs a deeper technical review. For now, will leave this change out of the commits.